### DISCUSSION TITLE
Why do the docs refer to unpublished versions?

### DISCUSSION THREAD

**Discussion by sfeldkamp** [babel/babel#16056] [Q&A] [answered]
For example, these two pages both refer to version 7.4, but the latest released version is 7.23.2.  There isn't even a version 7.4 in github releases or tags.

https://babeljs.io/docs/babel-preset-env#usebuiltins
https://babeljs.io/docs/babel-polyfill.html

And this page refers to versions 7.8, 7.11, and 7.13. 

https://babeljs.io/docs/options

Am I misunderstanding something about how or where to look for version numbers?  Or is this all a giant mess?

**Comment by nicolo-ribaudo** [comment, marked as answer]
23 comes after 4, so 7.23 comes after 7.4 (they are versions, not decimal numbers 😛)

You can find the full v7 changelog split between https://github.com/babel/babel/blob/main/CHANGELOG.md and https://github.com/babel/babel/blob/main/.github/CHANGELOG-v7.0.0-v7.14.9.md.

**Reply by sfeldkamp** [reply]
I wished so bad there was a delete option!  😆 

Thank you.
